lib/stack_master/change_set.rb

```diff
@@ -42,7 +42,9 @@ def initialize(describe_change_set_response)
     end
 
     def display(io)
-      io.puts "Proposed change set:"
+      io.puts "
```

Comment:
nitpick. Is this line break needed?

Reply:
I think it'll look better. Here's what it would look like without (new images added to PR description)

![screen shot 2018-02-21 at 3 05 42 pm](https://user-images.githubusercontent.com/2901882/36462684-f215c8fc-1718-11e8-93ae-35d6ec3cbe9a.png)